bocas/result.py

```diff
@@ -53,30 +59,67 @@ def get(self, name):
 
     @staticmethod
     def load(path):
-        with open(f"{path}/results.p", "rb") as f:
-            result = pickle.load(f)
+        # Maintain backwards compatibility with pickled results
+        if os.path.exists(f"{path}results.p"):
+            with open(f"{path}results.p", "rb") as f:
+                result = pickle.load(f)
+        elif os.path.exists(f"{path}results.yaml"):
+            with open(f"{path}results.yaml", "r") as f:
+                result = yaml.load(f, Loader=yaml.FullLoader)
+
         return result
 
     @staticmethod
     def load_collection(path):
         results = []
-        for path in glob.glob(f"{path}/*"):
+        for path in glob.glob(f"{path}/*/"):
```

Comment:
this restricts it to directories, which is perhaps what we want?  whats the intention here

Reply:
I think this is what we would want since the saving structure is in `dir/results.yaml` and not just `./results.yaml`.  A bonus about restricting to directories is that a readme can be included along with other files without the code breaking.  For example `./dir/README.md`.  What do you think?

Reply:
I'm not sure I'm understanding - you've updated the glob to be:

`{path}/*/` so they have to end in backslashes - so `results.yml` won't show up.  Is that incorrect?